src/ts/utils/SubtitleUtils.ts

```diff
@@ -4,7 +4,11 @@ import { PlayerAPI, SubtitleEvent, SubtitleTrack } from 'bitmovin-player';
 import { i18n } from '../localization/i18n';
 import { StorageUtils } from './StorageUtils';
 import { prefixCss } from '../components/DummyComponent';
-import { StoredSubtitleLanguage } from '../components/buttons/SubtitleToggleButton';
+
+export interface StoredSubtitleLanguage {
```

Comment:
Let's also remove the `setSubtitleLanguageStorage` for now. We can always add it again when we implement this.